spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-local.adoc

```diff
@@ -133,6 +133,14 @@ $ docker exec -it dataflow-server java -jar shell.jar
 
 Use of Spring Cloud Data Flow Shell is further described in <<shell,Shell>>.
 
+==== Spring Cloud Data Monitoring
+
+By default the Data Flow docker-compose configures Stream monitoring with InfluxDB and pre-build dashboards for Grafana.
+
+For further instructions about the Data Flow monitoring follow the <<streams-monitoring-local-influx,Streams Monitoring InfluxDB >> section.
```

Comment:
maybe - `instructions about Data Flow monitoring, follow the`

Reply:
+1